mass_mailing_custom_unsubscribe/readme/CONFIGURE.rst

```diff
@@ -5,3 +5,12 @@ they are going to unsubscribe. To do it:
 #. Create / edit / remove / sort as usual.
 #. If *Details required* is enabled, they will have to fill a text area to
    continue.
+
+For having discrete unsubscriptions from other recipients than the mailing
+lists, you need to add a glue module that adds 2 fields in the associated
+model:
+
+- `opt_out`.
+- Either `email` or `email_from`.
+
+See `mass_mailing_custom_unsubscribe_event` for an example.
```

Comment:
I think it makes sense for this module to provide an `AbstractModel` that makes these operations, so other modules such as `mass_mailing_custom_unsubscribe_event` only have to inherit it.

BTW, you're mixing syntaxes here, ans shouldn't the glue module be prefixed with the core addon it's based on? `mass_mailing_event_custom_unsubscribe`?:

```suggestion
- ``opt_out``.
- Either ``email`` or ``email_from``.

See ``mass_mailing_event_custom_unsubscribe`` for an example glue module
that adds this feature to events.
```

Reply:
The name is put that way for having all of them grouped together, as it depends on the main one, so that's the main factor. Even more, this module doesn't depend on `mass_mailing_event` on purpose, as it's not needed, as that one only adds some UI commodities.

Reply:
I'm also not providing any AbstractModel as being only 2 fields that can collapse with existing ones, so I prefer that people defines them as their taste. It's not adding a valuable piece of reusable code, taking into account that most of it will be even rewritten for adjusting it to the target model.